cmsBuild

```diff
@@ -3618,6 +3621,11 @@ def build(opts, args, factory):
         fatal("unable to find a working rpm.")
     rpmCacheUpdate(opts)
     packages = [factory.createWithSpec(pkgName) for pkgName in args]
+    if opts.depsLog:
+        from json import dumps as jdumps 
+        err, cmsdist_branch = getstatusoutput("cd %s; git rev-parse --abbrev-ref HEAD" % opts.cmsdist)
+        cmsdist_gh_url = "https://github.com/cms-sw/cmsdist/blob"
+        
```

Comment:
I would not do it in here. cmsdist might not always be a git based area. and we really do not need the urls in the deps file. The client which runs cmsBuild can add the urls if neded.